### PR TITLE
snyk: add page

### PR DESCRIPTION
**Pull request by darmalovan**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/snyk.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# snyk
+
+> Find vulnerabilities in your code and remediate risks.
+> More information: <https://snyk.io>.
+
+- Login to your snyk account:
```
Comment (pxgamer) [inline pages/common/snyk.md]:
```suggestion
- Login to your Snyk account:
```

File: pages/common/snyk.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`snyk auth`
+
+- Test your code for any known vulnerabilities:
+
+`snyk test`
+
+- Test a local docker image for any known vulnerabilities:
```
Comment (pxgamer) [inline pages/common/snyk.md]:
```suggestion
- Test a local Docker image for any known vulnerabilities:
```

**Comment by darmalovan** [comment]
Thanks @pxgamer Would be ok with me amending those changes?

**Comment by mebeim** [comment]
Thank you for the page @darmalovan :blush:

**Pull request merged**